utils.py

```diff
@@ -0,0 +1,257 @@
+                'currency': band.currency.symbol
+            }
+            data['tbody'][country_name][str(level)] = cell
+
+    return data
+
+
+def make_table_of_standard_supporters_by_country_and_size():
+    sizes = models.SupporterSize.objects.all().order_by('multiplier')
+    standard_level = get_standard_support_level()
+
+    data = {}
+    data['thead'] = []
+    for size in sizes:
+        data['thead'].append(str(size))
+
+    data['tbody'] = {}
+    for country, curr_code, region in iter_demo_countries():
+        currency, _ = models.Currency.objects.get_or_create(
+            code=curr_code,
+            region=region,
+        )
+        for size in sizes:
+            band = get_abstract_band(size, standard_level, country, currency)
+            country_name = abbreviate_us_uk(band.country)
+            if country_name not in data['tbody']:
+                data['tbody'][country_name] = {}
+            cell = {
+                'fee': band.fee,
+                'currency': band.currency.symbol
+            }
+            data['tbody'][country_name][str(size)] = cell
+
+    return data
+
+
+def make_table_showing_all_levels_by_country_and_size():
+    levels = models.SupportLevel.objects.all().order_by('-order')
+
+    data = {}
+    data['thead'] = []
+    for level in levels:
+        data['thead'].append(level.name)
+
+    data['tbody'] = {}
+    for size in models.SupporterSize.objects.all().order_by('multiplier'):
+        size_display = str(size)
+        data['tbody'][size_display] = {}
+        for country, curr_code, region in iter_demo_countries():
+            for level in levels:
+                currency, _ = models.Currency.objects.get_or_create(
+                    code=curr_code,
+                    region=region,
+                )
+                band = get_abstract_band(size, level, country, currency)
+                country_name = abbreviate_us_uk(band.country)
+                if country_name not in data['tbody'][size_display]:
+                    data['tbody'][size_display][country_name] = {}
+                cell = {
+                    'fee': band.fee,
+                    'currency': band.currency.symbol
+                }
+                data['tbody'][size_display][country_name][level.name] = cell
+
+    return data
+
+
+def generate_new_demo_data():
+    return [
+        make_table_of_higher_supporters_by_country_and_level(),
+        make_table_of_standard_supporters_by_country_and_size()
+    ]
+
+
+def update_demo_band_data():
+    data_json = json.dumps(generate_new_demo_data(), separators=(',', ':'))
+    filename = os.path.join(plugin_settings.SHORT_NAME, DEMO_DATA_FILENAME)
+    return save_media_file(filename, data_json)
+
+
+def get_saved_demo_band_data():
+    filename = os.path.join(plugin_settings.SHORT_NAME, DEMO_DATA_FILENAME)
+    file = cms_models.MediaFile.objects.get(label=filename)
+    with file.file.open('r') as file_ref:
+        return json.loads(file_ref.read())
```

Comment:
Once you move to using Decimal values, python's JSON library does not have codecs for `decima.Decimal`, you will need to either pass a codec or manually transform the values as you encode/decode